x-pack/metricbeat/module/redisenterprise/_meta/supported-versions.yml

```diff
@@ -0,0 +1,2 @@
+variants:
+  - REDIS_VERSION: 5.4.10-22
```

Comment:
Please use `REDISENTERPRISE_VERSION` for this variable, in case at some moment we want to have scenarios with different versions of redis and redis enterprise.

Reply:
Fixed.